common/models/api.js

```diff
@@ -6,7 +8,7 @@ module.exports = function api(Api) {
       isStatic: true,
       accepts: [],
       returns: {arg: 'response', type: 'object', root: true},
-      description: 'Shutdown this server.',
+      description: g.t('Shutdown this server.'),
```

Comment:
The unit test passed up to the point master passes locally on my OSX.

strong-mesh-models globalization depends on loopback/explorer globalization strategy which is open at the moment.